src/components/pages/poll-page/components/answers-sheet/AnswersSheet.tsx

```diff
@@ -64,213 +37,216 @@ export const getProgress = (answers: Answer[], questions: Question[]) => {
+        disciplineTeacherId,
+      );
+      setIsSendingStatus(SendingStatus.SUCCESS);
+    } catch (error) {
+      const message = getErrorMessage(error);
+      const errorMessage =
+        message === 'Value is wrong'
+          ? 'Текст повинен містити не менше 4 символів'
+          : message || 'Щось пішло не так, спробуй пізніше!';
+      toast.error('Помилка!', errorMessage);
+      setIsSendingStatus(SendingStatus.ERROR);
+    }
+  };
 
   return (
-    <div
-      className={
+    <Box
+      sx={
         sendingStatus === SendingStatus.SUCCESS
-          ? styles.successWrapper
-          : styles.wrapper
+          ? sxStyles.successWrapper
+          : sxStyles.wrapper
       }
     >
-      {sendingStatus === SendingStatus.LOADING ? (
-        <div className={styles.loaderWrapper}>
+      {sendingStatus === SendingStatus.LOADING && (
+        <Box sx={sxStyles.loaderWrapper}>
           <Progress />
-        </div>
-      ) : sendingStatus === SendingStatus.SUCCESS ? (
-        <div className={styles.wrapper}>
+        </Box>
+      )}
+      {sendingStatus === SendingStatus.SUCCESS && (
+        <Box sx={sxStyles.answersSavedWrapper}>
           <AnswersSaved />
-        </div>
-      ) : (
-        <>
-          <div
-            className={styles.toQuestionsList}
-            onClick={() => {
-              setQuestionsListStatus(true);
-            }}
-          >
-            <Box sx={sxStyles.chevronIcon}>
+        </Box>
+      )}
+      {sendingStatus != SendingStatus.LOADING &&
+        sendingStatus != SendingStatus.SUCCESS && (
+          <>
+            <Box
+              sx={sxStyles.toQuestionsList}
+              onClick={() => {
+                setQuestionsListOpened(true);
+              }}
+            >
               <ChevronLeftIcon style={{ height: '20px' }} />
+              <Typography sx={sxStyles.questionName}>
+                {currentCategory + 1} . {currentQuestions?.name}
+              </Typography>
             </Box>
-            <b>
-              {current + 1} . {category.name}
-            </b>
-          </div>
-          <div className={styles.answersWrapper}>
-            <Formik
-              validateOnMount
-              validateOnChange
-              initialValues={initialValues}
-              enableReinitialize
-              onSubmit={handleSubmit}
-            >
-              {({ values }) => (
-                <Form
-                  onClick={(event: FormEvent<HTMLFormElement>) => {
-                    // TODO: refactor this shit
-                    const name = (event.target as HTMLFormElement).name;
-                    const value = (event.target as HTMLFormElement).value;
-                    if (name && value) {
-                      values[name] = String(value);
-                      answer(values);
+            <Box sx={sxStyles.answersWrapper}>
+              <Formik
+                validationSchema={createValidationSchema}
```

Comment:
There is no argument provided, how does this validation schema work

Reply:
![image](https://github.com/fictadvisor/fictadvisor-web/assets/86970828/a179a6d2-a9e4-4f7d-b8f1-00aeee02faf5)
No, it doesn't, an error comes from the backend

Reply:
Outsorce creteValidattioinSchena in s separate file pls